pontoon/base/models.py

```diff
@@ -1335,6 +1340,12 @@ class Entity(DirtyFieldsMixin, models.Model):
     comment = models.TextField(blank=True)
     order = models.PositiveIntegerField(default=0)
     source = JSONField(blank=True, default=list)  # List of paths to source code files
+
+    # extra stores data that we want to save for the specific format
+    # this entity is stored in, but that we otherwise don't care
+    # about.
+    extra = JSONField(default=extra_default)
```

Comment:
(I didn't test it, but...) IMHO default=dict would create a new instance of dictionary. `default=lambda: {'foo':'bar'}` could be a solution... However, that's a small nit. I just thought that it's better to keep default values closer to the definition of model.